subsys/mgmt/mcumgr/grp/fs_mgmt/src/fs_mgmt.c

```diff
@@ -299,7 +299,6 @@ static int fs_mgmt_file_download(struct smp_streamer *ctxt)
 
 	/* Read the requested chunk from the file. */
 	bytes_read = fs_read(&fs_mgmt_ctxt.file, file_data, MCUMGR_GRP_FS_DL_CHUNK_SIZE);
-
```

Comment:
Put this line back

Reply:
Is that empty line not mistake?

Reply:
I mean I wouldn't call what's used in the vast majority of the zephyr tree strange, personally